app/museum/6529-fund-szn1/act-of-kindness/page.tsx

```
@@ -1,573 +1,565 @@
+    <link
+      rel="stylesheet"
+      id="genesis-blocks-style-css-css"
+      href="https://dnclu2fna0b2b.cloudfront.net/wp-content/plugins/genesis-blocks/dist/style-blocks.build.css?ver=1720746820"
+      type="text/css"
+      media="all"
+    />
+    <style
+      id="wp-emoji-styles-inline-css"
+      type="text/css"
+      dangerouslySetInnerHTML={{
+        __html:
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)